packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -322,19 +409,26 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
       return;
     }
     if (event is PointerMoveEvent || event is PointerPanZoomUpdateEvent) {
-      final Offset delta = (event is PointerMoveEvent) ? event.delta : (event as PointerPanZoomUpdateEvent).panDelta;
-      final Offset localDelta = (event is PointerMoveEvent) ? event.localDelta : (event as PointerPanZoomUpdateEvent).localPanDelta;
-      final Offset position = (event is PointerMoveEvent) ? event.position : (event.position + (event as PointerPanZoomUpdateEvent).pan);
-      final Offset localPosition = (event is PointerMoveEvent) ? event.localPosition : (event.localPosition + (event as PointerPanZoomUpdateEvent).localPan);
       if (_state == _DragState.accepted) {
-        _checkUpdate(
-          sourceTimeStamp: event.timeStamp,
-          delta: _getDeltaForDetails(localDelta),
-          primaryDelta: _getPrimaryValueFromOffset(localDelta),
-          globalPosition: position,
-          localPosition: localPosition,
-        );
+        _pointerMoveAccept = true;
+        // When some pointers come again, update immediately and add this event.
+        final bool eventExists = _multiPointerMoveTrackers.any((PointerEvent element) => element.pointer == event.pointer);
```

Comment:
>  `Map<int, PointerEvent>`

You mean that using the pointer as a key, and the PointerMoveEvent of the pointer as a value? If so, both Map and List are ok, but I do not think the Map is necessary. Because we do not store a sequence PointerMoveEvent of a pointer. What is your opinion?

Reply:
This logic needs to store the order to use in checkUpdate, `     sourceTimeStamp: _multiPointerMoveTrackers.last.timeStamp`. Therefore, it is more suitable to use List.